test/e2e/main_test.go

```diff
@@ -51,14 +51,22 @@ func main(m *testing.M) int {
 
 	exitCode := m.Run()
 
-	tests := []testing.InternalTest{{
-		Name: "NoReconcilationErrors",
-		F: func(t *testing.T) {
-			// see: https://github.com/rhobs/observability-operator/issues/200
-			t.Skip("skipping reconciliation error test until #200 is fixed")
-			f.AssertNoReconcileErrors(t)
+	tests := []testing.InternalTest{
+		{
+			Name: "NoReconcilationErrors",
+			F: func(t *testing.T) {
+				// see: https://github.com/rhobs/observability-operator/issues/200
+				t.Skip("skipping reconciliation error test until #200 is fixed")
+				f.AssertNoReconcileErrors(t)
+			},
 		},
-	}}
+		{
+			Name: "NoOwnerRefInvalidNamespaceReasonEvent",
+			F: func(t *testing.T) {
+				f.AssertNoEventWithReason(t, "OwnerRefInvalidNamespace")
```

Comment:
besides the reason, we might use some comment to recall what does this do in a while :D